packages/console/src/components/FormCard/index.tsx

```diff
@@ -23,7 +24,7 @@ function FormCard({ title, description, learnMoreLink, children }: Props) {
         <div className={styles.title}>{t(title)}</div>
         {description && (
           <div className={styles.description}>
-            {t(description)}
+            <DynamicT forKey={description} />
```

Comment:
When to use `t()` and when to use the new `DynamicT`?

Reply:
per pr description

Reply:
But why in line 24, the `t(title)` is not replaced by `<DynamicT />`, since the `title` is also a dynamic variable?

Reply:
i have no idea why it doesn't work for multiple children while it works for a single child. tried to check the types but they look the same

Reply:
will fix in the next PR